### PR TITLE
Add `promise-hook` community hook

### PR DESCRIPTION
**Pull request by aiven715**

### CODE REVIEW

File: public/docs/community/promise-hook.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    lastUpdated,
+    error
+  };
+};
+```
+
+# usage
+
+```javascript
+import React from "react";
```
Comment (vivek12345) [inline public/docs/community/promise-hook.md]: @aiven715 Thanks for contributing. For the demo on the app, we do not allow import statement's. So can you please remove the import and export statement's if present any where.

Comment (aiven715) [reply public/docs/community/promise-hook.md]: @vivek12345 yep, removed imports from the example

File: public/docs/community/promise-hook.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    lastUpdated,
+    error
+  };
+};
+```
+
+# usage
+
+```javascript
+const Movies = () => {
```
Comment (vivek12345) [inline public/docs/community/promise-hook.md]: @aiven715 for showing preview or demo of the hook, it has to be inside a component called Demo.
You can always refer to this template for example
https://github.com/Raathigesh/hooks.guide/blob/master/public/docs/template.md
or check other hooks inside public/docs directory.
Thanks

**Comment by vivek12345** [comment]
Looks good 🎉 Thanks for contributing @aiven715. Feel free to add yourself as a contributor if you would like to 👍

**Comment by aiven715** [comment]
@vivek12345 thanks! I've added myself to contributors list

File: .all-contributorsrc [hunk 3]
```diff
@@ -40,6 +40,15 @@
       "contributions": [
         "doc"
       ]
+    },
+    {
```
Comment (VivekNayyar) [inline .all-contributorsrc]: For this to reflect on the README, I think you will have to save and commit the updated README as well.

File: .all-contributorsrc [hunk 4]
```diff
@@ -40,6 +40,15 @@
...
+      "login": "aiven715",
```
Comment (vivek12345) [inline .all-contributorsrc]: For this to reflect on the README, I think you will have to save and commit the updated README as well.
@aiven715

Comment (aiven715) [reply .all-contributorsrc]: @vivek12345 thanks, I've updated readme

**Comment by vivek12345** [comment]
Thanks 💯

**Pull request merged**